app/lib/widgets/base_screens/base_list_screen.dart

```diff
@@ -58,7 +60,7 @@ class _BaseListScreenState extends State<BaseListScreen> {
 
     return Scaffold(
       appBar: AppBar(
-        backgroundColor: Theme.of(context).colorScheme.inversePrimary,
+        backgroundColor: Theme.of(context).colorScheme.surface,
```

Comment:
I do think it looks rather weird to have the appbar the same color as the rest of the surface.  

Also i think we should define a constant for the appbar color. Or a util method  like: 
```dart
AppBar buildAppBar(BuildContext context, String title,
        [VoidCallback? onReload]) =>
    AppBar(
        backgroundColor: Theme.of(context).colorScheme.surface,
        title: Text(title),
        actions: [
          if (onReload != null && kIsWeb)
            IconButton(onPressed: onReload, icon: const Icon(ManvIcons.refresh))
        ]);
```